.github/workflows/ci.yaml

```diff
@@ -94,6 +94,16 @@ jobs:
           -H "Accept: application/vnd.github.everest-preview+json" \
           -d '{"event_type": "honeybee_radiance_release", "client_payload": {"version": "${{ steps.new_release.outputs.tag }}"}}' \
           -u ladybugbot:$DEPS_TOKEN
+      - name: Update honeybee-radiance-postprocess
+        if: contains(steps.new_release.outputs.tag, '.')
+        env:
+          DISPATCH_REPO: ladybug-tools/honeybee-radiance-postprocess
+          DEPS_TOKEN: ${{ secrets.DEPS_UPDATING }}
+        run: |
+          curl -X POST https://api.github.com/repos/$DISPATCH_REPO/dispatches \
+          -H "Accept: application/vnd.github.everest-preview+json" \
+          -d '{"event_type": "honeybee-radiance-postprocess_release", "client_payload": {"version": "${{ steps.new_release.outputs.tag }}"}}' \
```

Comment:
The `"event_type"` should be `honeybee_radiance_release` in order to align with what you have here:

https://github.com/ladybug-tools/honeybee-radiance-postprocess/blob/master/.github/workflows/dependency-release.yaml#L10